services/webnn/ort/graph_builder_ort.h

```diff
@@ -156,8 +156,11 @@ class GraphBuilderOrt {
                       OperandDataType input_data_type,
                       const std::vector<uint32_t>& bias_dims);
 
-  // A helper function used to convert the weight or bias from Rzn to Zrn.
-  std::string ConvertRznToZrn(std::string_view weight_or_bias);
+  // A helper function used to reorder the weight and bias data for the RNN
+  // operators (GRU, LSTM, etc.).
```

Comment:
You may give examples of the usage, such as comment in your implementation, such as

// To reorder gru weight and bias from "rzn" layout to "rzn" layout, pass permutation as {1, 0, 2}.
// To reorder lstm weight and bais from "ifgo" layout to "iofg" layout, pass permutation as {0, 3, 1, 2}

Reply:
Done.